### PR TITLE
Add new motifs and furnishings introduced in update 34

### PR DESCRIPTION
**Pull request by MuMuQ**
The change also includes some style motifs that do not have a
matching achievement yet. Those are commented out for now
in order to avoid confusion. Considering the usual release pattern
of motifs those will probably be rewards for daily quests on High
Isles.

### CODE REVIEW

**Comment by MuMuQ** [comment]
One of the style that are currently commented out might be the raid style. I'll add the achievement id later tonight

**Comment by MuMuQ** [comment]
Ok. I increased the cap in the achievement finder and could not find any additional style achievements. The achievement subcategory for the raid does not contain the achievement either so the styles will probably start to drop after a few months.

TLDR: The PR contains all styles currently linked to an achievement.

**Comment by Rhyono** [comment]
The typical situation with the PTS: many styles appear, half without achievements. Once they go to live, the ones without achievements are deactivated. Then months later, they get introduced.
